src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -48,6 +48,18 @@
  * <pre>
  * &lt;module name=&quot;PackageAnnotation&quot;/&gt;
  * </pre>
+ * <p>
+ * Examples of validating files package-info.java and MyClass.java:
+ * </p>
+ * <pre>
+ * package-info.java     //ok
+ * &#64;Deprecated
+ * package com.example.annotations.packageannotation;
+ *
+ * MyClass.java   //violation
+ * &#64;Deprecated
+ * package com.example.annotations.packageannotation;
```

Comment:
this two lines should be in separate PRE tag

Reply:
Done